### DISCUSSION TITLE
如何针对某个文件夹加密，实现「输入密码后」才能访问？

### DISCUSSION THREAD

**Discussion by fajianren** [Studio-42/elFinder#3639] [Q&A]
步骤：
1. 用户访问一个加密的文件
2. 输入正确的密码
3. 展示该文件夹下的文件，并可下载

**Comment by h3786010** [comment]
作法一：在elFinder外(需要DB)
1.開發管理頁面:
1-1.輸入密碼...等資料。
1-2.開窗elFinder→選擇1個物件(檔案or資料夾)→將hash取回。
1-3.儲存DB。
2.開發下載頁面:
2-1.透過分享網址進入。
2-2.驗證密碼...等資料。
2-3.開啟分享elFinder指定hash(若不想直接開放存取，可複製一份到其他目錄...較安全但要注意複製的檔案大小)

作法二：在elFinder內(需要DB)
https://github.com/Studio-42/elFinder/issues/3109

作法三：只有複製分享連結
https://github.com/Studio-42/elFinder/issues/2905
